Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -621,6 +631,9 @@ bool consumeStyleProperty(CSSParserTokenRange& range, const CSSParserContext& co
             return true;
         }
 
+        if (consumeCSSFunctionValue(range, property, state, result))
+            return true;
```

Comment:
I think this call can be earlier, before `if (WebCore::isShorthand(property))` test so you don't need two call sites.

Reply:
This statement was moved and it is now called once.